spec/inputs/custom_input_spec.rb

```diff
@@ -11,14 +11,16 @@ def input_args
     @method = :title
     @options = {}
     @proc = Proc.new {}
+
     if Rails::VERSION::MAJOR == 4
       @builder = Formtastic::FormBuilder.new(@object_name, @object, @template, @options)
-    else
-      @builder = Formtastic::FormBuilder.new(@object_name, @object, @template, @options, @proc)
+      return [@builder, @template, @object, @object_name, @method, @options]
```

Comment:
this whole thing will be improved by removing the condition by https://github.com/justinfrench/formtastic/pull/1188/commits/b51d06d92f4f80560951ed8e5438a32c6f1fc487 (#1188)